src/store/store.js

```diff
@@ -1,6 +1,6 @@
 import { configureStore } from '@reduxjs/toolkit';
 import authReducer from './auth/Slice';
-import sampleReducer from './sampleFeature/slice';
+import sampleReducer from './samplefeat/slice';
 
 export const store = configureStore({
   reducer: {
```

Comment:
make sure to add the **users** slice inside the configureStore object. to be able to do something like **state.usersSlice.allUsers** when using the **useSelector** hook.